### ISSUE TITLE
Results example blocks are not being exported from org-mode to markdown

### ISSUE THREAD

**Issue by e-eight** [kaushalmodi/ox-hugo#424]
## Actual Behavior
In my source org-mode file I have linked some external org-mode files as subtrees.  The external org-mode files have some `#+begin_example ... #+end_example` blocks. These are the outputs of some python code being run in a `src` block. While the `src` block in correctly exported to the markdown file, the results in the `example` block are not exported at all. However if I put a 
```
#+RESULTS:
#+begin_example
1234
#+end_example
```
in the central source file itself then that exports correctly.

## Expected Behavior
Results `example` blocks of external files are exported to markdown.

### Example Org File
```
source.org

:PROPERTIES:
:EXPORT_FILE_NAME: external
:END:
#+INCLUDE: "external.org"
```
```
external.org

#+begin_src jupyter-python
lm_adam = LinearModel()
lm_adam.to(device)
optimizer = Adam(lm_adam.parameters(), weight_decay=0.0001)

for epoch in range(epochs):
    running_loss = 0.0

    for i, (x, y) in enumerate(dummy_data):

        x_ = Variable(x, requires_grad=True)
        y_ = Variable(y)

        # Forward pass
        y_pred = lm_adam(x_)

        # Compute loss
        loss = criterion(y_pred, y_)

        # Zero gradients, backward pass, and update weights
        optimizer.zero_grad()
        loss.backward()
        optimizer.step()

        # Update the running loss
        running_loss += loss.item()

    print(f"Epoch: {epoch + 1:02}/{epochs} Loss: {running_loss:.5e}")
#+end_src

#+RESULTS:
#+begin_example
Epoch: 01/20 Loss: 1.35205e+03
Epoch: 02/20 Loss: 1.33684e+03
Epoch: 03/20 Loss: 1.32229e+03
Epoch: 04/20 Loss: 1.30802e+03
Epoch: 05/20 Loss: 1.29396e+03
Epoch: 06/20 Loss: 1.28007e+03
Epoch: 07/20 Loss: 1.26634e+03
Epoch: 08/20 Loss: 1.25274e+03
Epoch: 09/20 Loss: 1.23927e+03
Epoch: 10/20 Loss: 1.22593e+03
Epoch: 11/20 Loss: 1.21272e+03
Epoch: 12/20 Loss: 1.19962e+03
Epoch: 13/20 Loss: 1.18664e+03
Epoch: 14/20 Loss: 1.17378e+03
Epoch: 15/20 Loss: 1.16103e+03
Epoch: 16/20 Loss: 1.14839e+03
Epoch: 17/20 Loss: 1.13587e+03
Epoch: 18/20 Loss: 1.12345e+03
Epoch: 19/20 Loss: 1.11114e+03
Epoch: 20/20 Loss: 1.09893e+03
#+end_example
```

### Generated Markdown File or Error
```
external.md

```jupyter-python
lm_adam = LinearModel()
lm_adam.to(device)
optimizer = Adam(lm_adam.parameters(), weight_decay=0.0001)

for epoch in range(epochs):
    running_loss = 0.0

    for i, (x, y) in enumerate(dummy_data):

        x_ = Variable(x, requires_grad=True)
        y_ = Variable(y)

        # Forward pass
        y_pred = lm_adam(x_)

        # Compute loss
        loss = criterion(y_pred, y_)

        # Zero gradients, backward pass, and update weights
        optimizer.zero_grad()
        loss.backward()
        optimizer.step()

        # Update the running loss
        running_loss += loss.item()

    print(f"Epoch: {epoch + 1:02}/{epochs} Loss: {running_loss:.5e}")
```
```

## Ox-Hugo Debug Information

<!--

  (Step 1) Run `M-x org-hugo-debug-info`.

     *That will copy the debug info in Markdown format to the kill ring.*
     So do *not* manually do any other kill (copy/cut) operation after that!

  See step 2 below.
-->

<details>
  <summary>Debug Info</summary>

  <!--
      (Step 2) Paste the Markdown contents (copied to kill-ring in
               Step 1 above)  after `</summary>` and before
               `</details>` below.
  -->


# Debug information for `ox-hugo`


## Emacs Version

    GNU Emacs 27.1 (build 1, x86_64-unknown-linux-gnu, GTK+ Version 2.24.32)
     of 2020-11-29


## Org Version

    Org mode version 9.5 (nil @ /home/soham/.emacs.d/.local/straight/build-27.1/org-mode/)


## Hugo Version

    Hugo Static Site Generator v0.80.0 linux/amd64 BuildDate: unknown


### Org `load-path` shadows

No Org mode shadows found in `load-path`


## `ox-hugo` defcustoms

<table border="2" cellspacing="0" cellpadding="6" rules="groups" frame="hsides">


<colgroup>
<col  class="org-left" />

<col  class="org-left" />
</colgroup>
<tbody>
<tr>
<td class="org-left">org-hugo-section</td>
<td class="org-left">&ldquo;posts&rdquo;</td>
</tr>


<tr>
<td class="org-left">org-hugo-use-code-for-kbd</td>
<td class="org-left">nil</td>
</tr>


<tr>
<td class="org-left">org-hugo-preserve-filling</td>
<td class="org-left">t</td>
</tr>


<tr>
<td class="org-left">org-hugo-delete-trailing-ws</td>
<td class="org-left">t</td>
</tr>


<tr>
<td class="org-left">org-hugo-prefer-hyphen-in-tags</td>
<td class="org-left">t</td>
</tr>


<tr>
<td class="org-left">org-hugo-allow-spaces-in-tags</td>
<td class="org-left">t</td>
</tr>


<tr>
<td class="org-left">org-hugo-tag-processing-functions</td>
<td class="org-left">(org-hugo&#x2013;tag-processing-fn-replace-with-spaces-maybe org-hugo&#x2013;tag-processing-fn-replace-with-hyphens-maybe)</td>
</tr>


<tr>
<td class="org-left">org-hugo-auto-set-lastmod</td>
<td class="org-left">nil</td>
</tr>


<tr>
<td class="org-left">org-hugo-export-with-toc</td>
<td class="org-left">nil</td>
</tr>


<tr>
<td class="org-left">org-hugo-export-with-section-numbers</td>
<td class="org-left">nil</td>
</tr>


<tr>
<td class="org-left">org-hugo-front-matter-format</td>
<td class="org-left">&ldquo;toml&rdquo;</td>
</tr>


<tr>
<td class="org-left">org-hugo-default-static-subdirectory-for-externals</td>
<td class="org-left">&ldquo;ox-hugo&rdquo;</td>
</tr>


<tr>
<td class="org-left">org-hugo-external-file-extensions-allowed-for-copying</td>
<td class="org-left">(&ldquo;jpg&rdquo; &ldquo;jpeg&rdquo; &ldquo;tiff&rdquo; &ldquo;png&rdquo; &ldquo;svg&rdquo; &ldquo;gif&rdquo; &ldquo;mp4&rdquo; &ldquo;pdf&rdquo; &ldquo;odt&rdquo; &ldquo;doc&rdquo; &ldquo;ppt&rdquo; &ldquo;xls&rdquo; &ldquo;docx&rdquo; &ldquo;pptx&rdquo; &ldquo;xlsx&rdquo;)</td>
</tr>


<tr>
<td class="org-left">org-hugo-date-format</td>
<td class="org-left">&ldquo;%Y-%m-%dT%T%z&rdquo;</td>
</tr>


<tr>
<td class="org-left">org-hugo-paired-shortcodes</td>
<td class="org-left">&ldquo;&rdquo;</td>
</tr>


<tr>
<td class="org-left">org-hugo-langs-no-descr-in-code-fences</td>
<td class="org-left">nil</td>
</tr>


<tr>
<td class="org-left">org-hugo-suppress-lastmod-period</td>
<td class="org-left">0.0</td>
</tr>


<tr>
<td class="org-left">org-hugo-front-matter-format</td>
<td class="org-left">&ldquo;toml&rdquo;</td>
</tr>
</tbody>
</table>


</details>

**Comment by kaushalmodi** [comment]
@e-eight Hello! Thanks for that recipe to reproduce the issue.

I confirm that I see that same thing as you are seeing. But I believe this is out of control of `ox-hugo`. I am seeing the same behavior is I export that subtree (`C-c C-e C-s ..`) using the inbuilt `ox-md` or `ox-html` exporters.

Can you reach out to the Org mailing list for help on this?

**Comment by e-eight** [comment]
`:exports both` worked. Thanks!
